lib/validate/clusterconfig.go

```diff
@@ -94,3 +99,28 @@ func ClusterConfiguration(existing, update clusterconfig.Interface) error {
 func isCloudConfigEmpty(global clusterconfig.Global) bool {
 	return global.CloudProvider == "" && global.CloudConfig == ""
 }
+
+// validateAdmissionPlugins verifies that the provided Kubernetes admission
+// plugins are valid.
+func validateAdmissionPlugins(plugins []string) error {
```

Comment:
nit: is modifying source code the only way to enable additional plugins? I'm not sure this is in scope, but kubernetes [has](https://github.com/kubernetes/kubernetes/blob/v1.17.9/pkg/kubeapiserver/options/plugins.go#L61) a list of all admission plugins which could be extracted/used for validation.